k2/csrc/ragged.h

```diff
@@ -338,6 +338,53 @@ struct Ragged {
 template <typename T>
 std::ostream &operator<<(std::ostream &stream, const Ragged<T> &r);
 
+/*
+  Stack a list of Ragged arrays to create a Ragged array with one more axis.
```

Comment:
Thanks, can you please remove the `const` from Ragged and RaggedShape inputs?  I want to get out of the habit of using that.  Getting row-ids/row-splits is not always const.

Reply:
Done in PR https://github.com/k2-fsa/k2/pull/250